code/index.html

```diff
@@ -1,21 +1,166 @@
 <!DOCTYPE html>
 <html>
-
   <head>
+    <link rel="icon" href="./favicon.ico" style="font-size: large" />
+    <script
+      src="https://kit.fontawesome.com/c38d857281.js"
+      crossorigin="anonymous"
+    ></script>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>AT Reinigung</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Montserrat&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <!-- 
-      Write your code here inside the body to make it appear in the browser. 
-      Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
-
-      <img src="example-recipe-background.png" alt="Banana Bread">
+    <div class="container2">
+      <div class="logo">
+        <img src="logo2.png" alt="logo" />
+      </div>
+    </div>
+    <nav>
+      <a href="./index.html">Home</a>
+      <a href="#services">Services</a>
+      <a href="#contact">Contact us</a>
+      <a class="appointment-button" href="mailto:info@atreinigung.ch"
+        >Book now</a
+      >
+    </nav>
+    <main>
+      <div class="hero-image">
+        <div class="hero-text">
+          <h1>Let us do the dirty job!</h1>
+          <h2>home and office cleaning services</h2>
+          <button>
+            <a style="color: white" ; href="tel: 0798704217" ;
+              >Book an appointment</a
+            >
+          </button>
+        </div>
+      </div>
+      <div id="services">
+        <div class="container">
+          <img
+            class="image"
+            src="domestic cleaning.jpg"
+            alt="domestic-cleaning"
+          />
+          <div class="text-overlay">Domestic cleaning</div>
+        </div>
+        <div class="container">
+          <img class="image" src="office cleaning.jpg" alt="office-cleaning" />
+          <div class="text-overlay">Office cleaning</div>
+        </div>
+        <div class="container">
+          <img
+            class="image"
+            src="windows cleaning.jpg"
+            alt="windows-cleaning"
+          />
+          <div class="text-overlay">Windows cleaning</div>
+        </div>
+      </div>
 
-    -->
-    <h1>Hello there! 👋🏼</h1>
+      <div class="icons">
```

Comment:
You could wrap the icon divs into a parent div and apply `display: flex` in the parent. This would make it easier in terms of responsiveness.
```
<div class="icons-wrapper">
  <div class="icons">...</div>
  <div class="icons">...</div>
  <div class="icons">...</div>
</div>
```